static/js/sandbox-main.js

```diff
@@ -48,3 +48,138 @@ mobileNavTriggerButton.addEventListener('click', function () {
+      appID.type = 'password';
+      appSecret.type = 'password';
+    }
+  });
+});
+
+// Toggle for Keyboard Naviagation
+appCredentialToggleLinks.forEach((appCredentialToggleLink) => {
+  appCredentialToggleLink.addEventListener('keypress', function (e) {
+    var key = e.which || e.keyCode;
+    if (key === 13) { // 13 is enter
+      let appID = document.querySelector('#id-' + this.id);
+      let appSecret = document.querySelector('#secret-' + this.id);
+      if (appID.type == 'password') {
+        appID.type = 'text';
+        appSecret.type = 'text';
+      } else {
+        appID.type = 'password';
+        appSecret.type = 'password';
+      }
+    }
+  });
+});
+
+// Copy Credentials to Clipboard
+function copyCredential(copyID) {
+  // Determine if a user is copying a client id or secret
+  if (copyID.includes('secret') === true) {
+    // Get the element we want to copy
+    let secretToCopy = document.querySelector('#' + copyID);
+    // If it is hidden, change to text, copy, and then hide again
+    if (secretToCopy.type == 'password') {
+      secretToCopy.type = 'text';
+      secretToCopy.select();
+      document.execCommand('copy');
+      secretToCopy.type = 'password';
+    } else {
+      // Else, if it is shown, just copy!
+      secretToCopy.select();
+      document.execCommand('copy');
+    }
+    // Fade In / Out Copy Confirmation Message
+    let copyConfirmation = document.querySelector('#confirm-' + copyID);
+    $( copyConfirmation ).fadeIn( 300 ).delay( 1500 ).fadeOut( 400 );
+  } else {
+    // Get the element we want to copy
+    let IDToCopy = document.querySelector('#' + copyID);
+    // If it is hidden, change to text, copy, and then hide again
+    if (IDToCopy.type == 'password') {
+      IDToCopy.type = 'text';
+      IDToCopy.select();
+      document.execCommand('copy');
+      IDToCopy.type = 'password';
+  } else {
+      // Else, if it is shown, just copy!
+      IDToCopy.select();
+      document.execCommand("copy");
+   }
+   // Fade In / Out Copy Confirmation Message
+   let copyConfirmation = document.querySelector('#confirm-' + copyID);
+   $( copyConfirmation ).fadeIn( 300 ).delay( 1500 ).fadeOut( 400 );
+  }
+};
+
+// BB Accordion Component
+// ----------------------
+// Used for: Sandbox Application Registration - Optional Info Accordion
+
+const accordions = document.querySelectorAll('.accordion-toggle');
+
+accordions.forEach(accordion => {
+  accordion.addEventListener('click', function() {
+    let contentToToggle = this.nextElementSibling;
+    let iconToToggle = this.querySelector('.accordion-icon');
+
+    if (contentToToggle.classList.contains('accordion-content-visible')) {
+      contentToToggle.classList.remove('accordion-content-visible');  
+      iconToToggle.innerHTML = feather.icons['chevron-down'].toSvg();
+    } else {
+      contentToToggle.classList.add('accordion-content-visible');
+      iconToToggle.innerHTML = feather.icons['chevron-up'].toSvg();
+    }
```

Comment:
Same comment as the secret toggle.

Reply:
Added a commit with the internal logic separated, just like we did with the secret toggle.